src/eth/AccountsService.js

```diff
@@ -73,12 +74,15 @@ export default class AccountsService extends PublicService {
       this.get('log').info(`Not adding account "${name}" (no address found)`);
       return;
     }
+    accountData.address = accountData.address.toLowerCase();
+
     if (this._getAccountWithAddress(accountData.address)) {
       throw new Error('An account with this address already exists.');
     }
 
     if (!name) name = accountData.address;
     const account = { name, type, ...accountData };
+    if (otherSettings.autoSwitch) account.autoSwitch = otherSettings.autoSwitch;
```

Comment:
I did it this way so that if `autoSwitch` isn't provided (because it's optional), the account object returned won't contain an `autoSwitch` property that's `undefined`, e.g.:
```js
{
  name: "0x72776bb917751225d24c07d0663b3780b2ada67c",
  type: "browser",
  autoSwitch: undefined,
  subprovider: ProviderSubprovider,
  address: "0x72776bb917751225d24c07d0663b3780b2ada67c"
}
```

(Since `autoSwitch` only applies to browser account types, the flag wouldn't be relevant for other account types.)

Alternatively, we can be explicit about it and just always return it with a default value of `false`.